src/Shared/Recruit.Vacancies.Client/Application/Constants.cs

```diff
@@ -3,5 +3,7 @@ namespace Esfa.Recruit.Vacancies.Client.Application
     public static class Constants
     {
         public const string HelpdeskPhoneNumber = "08000 150 600";
+        public const long EsfaTestTrainingProviderUkprn = 10033670;
```

Comment:
Rather than these constants, could you not use the `EsfaTestTrainingProvider` which already has these values as constants?

Reply:
Oops. Missed we already had this class 👍